examples/scripts/ct_astra_3d_tv_admm.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+3D TV-Regularized Sparse-View CT Reconstruction
+============================================
+
+This example demonstrates solution of a sparse-view CT reconstruction
+problem with isotropic total variation (TV) regularization
+
+  $$\mathrm{argmin}_{\mathbf{x}} \; (1/2) \| \mathbf{y} - A \mathbf{x}
+  \|_2^2 + \lambda \| C \mathbf{x} \|_{2,1} \;,$$
+
+where $A$ is the Radon transform, $\mathbf{y}$ is the sinogram, $C$ is
+a 3D finite difference operator, and $\mathbf{x}$ is the desired
+image.
+"""
+
+import numpy as np
+
+import jax
+
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from scico import functional, linop, loss, metric, plot
+from scico.linop.radon_astra import TomographicProjector
+from scico.optimize.admm import ADMM, LinearSubproblemSolver
+from scico.util import device_info
+
+
+def make_tangle(nx, ny, nz):
```

Comment:
Perhaps worth moving this example into `scico/examples.py`? ("No" is an acceptable answer!)